charts/grafana/FLUX.md

```diff
@@ -1,12 +1,32 @@
 # Flux language setup
-### Manual Setup
-* Flux is still in beta, so this information may be out of date
-* Follow instructions laid out by [Flux Datasource for Grafana]
-(https://grafana.com/plugins/grafana-influxdb-flux-datasource)
+## Manual Setup
+- [Flux](https://docs.influxdata.com/flux/v0.12/) is still in beta, so this information may be out of date
+- [Enable](https://docs.influxdata.com/flux/v0.12/introduction/installation) Flux by setting the `flux-enabled` option to `true` under the `http` section of your influxdb.conf
+- Follow instructions laid out by [Flux Datasource for Grafana](https://grafana.com/plugins/grafana-influxdb-flux-datasource)
 
+## Cronus-Monitoring Setup
+- Run the init.sh script in the scripts directory and the included helm chart will enable flux automatically
+    > Flux is enabled in Grafana via the [values.yaml](https://github.com/Eliforbes42/cronus-monitoring/blob/master/charts/grafana/values.yaml) `datasources` with `grafana-influxdb-flux-datasource` as the type
```

Comment:
This needs correct spacing. It merges together in the real document.

Reply:
I fixed this as well, changed both of those to bullets instead as they were a clarification and less of a note